CoEpi/repo/SymptomRepo.swift

```diff
@@ -44,11 +44,10 @@ class SymptomRepoImpl: SymptomRepo {
 }
 
 private extension Sequence where Iterator.Element == Symptom {
-
     func toCENReport() -> CenReport? {
-        // TODO (has not been specified yet)
-        "TODO symptoms -> CENReport".toBase64().map {
-            CenReport(id: "123", report: $0, timestamp: Date().coEpiTimestamp)
-        }
+        let separator = ", "
+        let stringReport : String  = self.reduce("") {$0 + $1.name + separator}
```

Comment:
You can use `map { $0.name }.joined(separator: ", ")`.  Then you don't need `trimmingCharacters`.
`self` not needed here.